packages/firebase_data_connect/firebase_data_connect/example/dataconnect/dataconnect.yaml

```diff
@@ -0,0 +1,11 @@
+specVersion: "v1alpha"
+serviceId: "dataconnect"
+location: "us-west2"
+schema:
+  source: "./schema"
+  datasource:
+    postgresql:
+      database: "dataconnect"
+      cloudSql:
+        instanceId: "fdc-test-north1"
```

Comment:
Same thing here, does it mean that the user needs to have an instanceId defined like this one?

Reply:
Only when deploying to production